### ISSUE TITLE
read_csv with "chunksize" does not forward pandas_kwargs "encoding" 

### ISSUE THREAD

**Issue by meganburger** [awslabs/aws-data-wrangler#330]
```
df_itr = awswrangler.s3.read_csv(
        s3_input_path,
        chunksize=500000,
        encoding='latin'
    )
```

gives the error

```
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xad in position 191537: invalid start byte
```

However, when I take out "chunksize" the read-in is successful. I looks like this parameter does not forward with "chunksize" read-in.

**Comment by igorborgest** [comment]
Hi @meganburger, thanks for reporting it.

We fixed it in the PR #328 

Could you test our dev branch and check if it addresses your use case before the official release?
The intention is to release it in the version `1.7.0` later this week.

```
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by meganburger** [comment]
Yes that worked for me, thank you!

**Comment by igorborgest** [comment]
Released in [1.7.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.7.0)!

**Comment by igorborgest** [comment]
Hi @meganburger !

It is totally off-topic, but we are stating a [_"Who uses AWS Data Wrangler?"_](https://github.com/awslabs/aws-data-wrangler/tree/dev#who-uses-aws-data-wrangler) section. So feel free to add yourself if you want 😄 .
